commands/deploy.go

```diff
@@ -44,7 +46,8 @@ func DeployCommand() cli.Command {
 }
 
 type deploycmd struct {
-	clientV2 *v2Client.Fn
+	clientV2      *v2Client.Fn
```

Comment:
Do we still need clientV2? Does apiClientAdapter replace it?

Reply:
We do need it now since not all references to it have been removed. As the implementation is fleshed out, we will be able to move it from this struct and many others.

Reply:
Yes I suppose we'll want to clean this up before this is merged (to keep things clean).